apps/sertra/components/mod-selector/modal.module.css

```diff
@@ -25,66 +25,64 @@
 }
 
 .header {
-  background-color: var(--bg-light);
   display: flex;
   flex-direction: column;
-  padding: 24px 32px 16px;
+  padding: 1.5rem 2rem 1rem;
+  background-color: var(--bg-light);
 }
 
 .headerTitle {
   display: flex;
-  flex-direction: row;
+  flex-direction: row-reverse;
   align-items: center;
   justify-content: space-between;
+  margin-bottom: 1rem;
+}
+
+.headerTitle:last-child {
+  margin-bottom: 0;
 }
 
 .headerTitle svg {
   width: 1.375rem;
   height: 1.375rem;
 }
 
-.headerContent {
-  margin-top: 16px;
-}
-
 .content {
+  height: 38rem;
```

Comment:
This works for this alpha release, but we should consider resizing to a different sized screens.

An example what we could do here: Set padding to the root parent, set the modal width and height to 100% with max constraints, remove explicit height from the content area and have it take up the remaining space automatically with `flex:1`.

Reply:
Agreed, and I listed this as one of the tasks Oksamies could to on Friday if he had the time. (The height was already fixed, this PR only changed it from `px` to `rem`.)